src/GlobalPayments.Api/Terminals/INGENICO/Responses/CancelResponse.cs

```diff
@@ -0,0 +1,18 @@
+﻿using GlobalPayments.Api.Terminals.Abstractions;
+using GlobalPayments.Api.Utils;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GlobalPayments.Api.Terminals.Ingenico {
+    public class CancelResponse : IngenicoTerminalResponse, IDeviceResponse{
```

Comment:
I maintain this is not needed. The only "response" is success/failure and was handled just fine by the base DeviceResponse object. It either works or throws an exception.